ModManager/ModSystem/Manifest.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Modio.Models;
+using Newtonsoft.Json;
+
+namespace ModManager.ModSystem
+{
+    public class Manifest
+    {
+        public static readonly string FileName = "manifest.json";
```

Comment:
Doesn't matter too much, but for extra safety `static readonly` could be changed to `const`